pyuvsim/tests/test_simsetup.py

```diff
@@ -119,10 +118,12 @@ def test_param_flux_cuts():
                                                 + [astropy.io.votable.exceptions.W50] * 10)
     catalog = pyuvsim.simsetup.array_to_skymodel(catalog)
     for sI in catalog.stokes[0, :]:
-        nt.assert_true(np.all(0.2 < sI < 1.5))
+        assert np.all(0.2 < sI < 1.5)
 
 
-def check_param_reader(config_num):
+# parametrize will loop over all the give values
+@pytest.mark.parametrize("config_num", list(range(6)))
```

Comment:
this is cool!

Reply:
Cool as this is... I don't think it's necessary anymore to run tests for all these different configuration files. Most of the other cases covered by the tests for `parse_frequency_params` and `parse_time_params`. That's why I changed it to just run the first config.

I guess it doesn't really hurt to run more tests, except to slow down travis. We should check more closely to see which configuration cases are still needed.

Reply:
Hm it did not seem obvious that these test cases were not being run anymore, in fact a new test case was added during one of my re-bases (e.g. there was an `if config_num==5` area).

Reply:
Odd.. I don't know when that was introduced. Currently on master, the `test_param_reader` function loops over the list `[0]`, so it's only running the zeroth case. The coverage is the same, implying some testing redundancy.

Reply:
interesting. I'll look to see if those are indeed not needed.

Reply:
Thanks. I probably should have cleaned that up myself before merging that change, but these cases were tricky to define and I'm nervous about removing them unless we're sure they're not needed anymore.

Reply:
looks like we gain 1 additional line of coverage by including config_num=2. let me track that down

Reply:
looks like in `config_num=2` we get also get this line at simsetup.py line 1458
```
      if par == 'sig': 

            return AnalyticBeam(model, sigma=float(val)) 
```